qr-code/src/main/java/com/adyen/checkout/qrcode/internal/ui/DefaultQRCodeDelegate.kt

```diff
@@ -153,6 +153,10 @@ internal class DefaultQRCodeDelegate(
         }
 
         val viewType = when (action.paymentMethodType) {
+            PaymentMethodTypes.DUIT_NOW -> {
```

Comment:
This when block will get even more polluted when PromptPay is added. I suggest creating an enum that holds `maxPollingDuration`, `componentViewType`, and also `paymentMethodType` with a helper function that maps from `paymentMethodType` to this enum and we'll read these constants directly from there. It'll be less work when we add new payment methods that fall into this flow.